service/templates/projects.html

```diff
@@ -470,17 +470,60 @@ <h2> Settings for [[displayName]]</h2>
           </div>
 
           <!--Github Commit Checks-->
-          <div class="form-group" ng-hide="true">
+          <div class="form-group">
             <div class="col-header col-lg-6 form-control-static">
               <h3>Github Commit Checks</h3>
             </div>
           </div>
-          <div id="github_checks" class="form-group" ng-hide="true">
+          <div id="github_checks" class="form-group">
             <div class="col-lg-6">
               <input type="checkbox" id="github-checks-checkbox" ng-model="settingsFormData.github_checks_enabled" />
               <label for="github-checks-checkbox">Enable Github Commit Checks</label>
             </div>
           </div>
+          <div ng-show="settingsFormData.github_checks_enabled === true && githubChecksConflicts.length === 0">
```

Comment:
Could you add some help text here on what the settings below do?

Reply:
Added, let me know if the wording is awkward

Reply:
![image](https://user-images.githubusercontent.com/26798134/103690339-6e7fac80-4f49-11eb-800b-1c3c6de1bd29.png)

Reply:
Could you avoid using the word waterfall since the new mainline UI is not called that? Maybe something like "When checked, commits will send their status as a Github Check when finished"